build/fastlane/Gemfile.lock

```diff
@@ -0,0 +1,218 @@
+GEM
```

Comment:
Should this file be excluded from source control? It seems generated.

Reply:
This ensures that you have the same versions all the time and that it don't breaks because of a new version of a dependency.

Reply:
I removed the file.